docs/website/contents/for-developers/HardWonWisdom.md

```diff
@@ -210,31 +210,41 @@ The possibility of an never-ending era is only supported for tests and for reuse
 
 In general, it would be possible to design a system such that even the final ledger state in an era did not determine when that era would end.
 That'd be an empty safe zone, consisting of zero slots.
 For example, perhaps the first block of an era identifies itself as such simply by setting some field in its header.
 For that kind of chain, some ledger states would not provide enough information to do any translations whatsoever of slots/times/epochs _after_ that ledger state's tip.
+(Note that if the first block of an era declares itself as such, then the node that minted that block has some how already decideded to transition to the next era.
+Hence the correspondingly hypothetical leadership check would have known whether it's about to forge a block in the current era or the next, and so it would still be able to determine the slot of the wallclock _according to the era of the block it's planning to forge_.)
 
-For the case of Cardano, however, other concerns (not directly related to slot/epoch/time translations) already require a seperate cutoff, similar to the horizon, to be above some minimum.
-For the sake of simplicity, we take that same minimum to be the safe zone.
-As a result, the safe zone is equal to one _stability window_, which is `2k` slots for Byron and `3k/f` slots for every Shelley era.
+For the case of Cardano, however, other concerns (not directly related to slot/epoch/time translations) already require a degree of "stability" from the ledger rules to be above some minimum.
+This "stability" is a stronger requirement than the safe zone, so, for the sake of simplicity, we take that same minimum (aka one _stability window_) to be the safe zone.
+Thus the safe zone is `2k` slots for Byron and `3k/f` slots for every Shelley era.
+
+Recap:
+
+- Consider some Cardano ledger state.
+  There is a first era that has an unknown end time according to that ledger state, which is also the last era with a known start time.
+- Even though the end time is unknown, it cannot be sooner than one safe zone after the known start time.
+- Moreover, if the ledger state also inhabits this era, the era cannot end sooner than one safe zone after the slot this ledger state was most recently ticked to.
+- Overall, it can be defined as: the _safe zone_ of an era is a lower bound on the number of slots that exist before the end of that era and after the latest-possible ledger state that does not determine the end of that era.
 
 ### Aside: The Necessary Stability of Header Validity
 
-[Esgen suggests that this could eventually be migrated to section 24.5 ("Eliminating safe zones") of the Consensus and Storage Layour Report (or the succeeding section).]
+[Esgen suggests that this could eventually be migrated to section 24.5 ("Eliminating safe zones") of the Consensus and Storage Layer Report (or the succeeding section).]
```

Comment:
Perhaps link to who this Esgen guy is :stuck_out_tongue_closed_eyes: or just omit this part?